app/containers/ShowActiveKey/saga.js

```diff
@@ -37,6 +59,70 @@ export function* showActiveKeyWorker({ resetForm, password }) {
   }
 }
 
+export function* sendEmailWorker() {
```

Comment:
Maybe call it `sendFacebookEmailWorker`?

Reply:
I agree with you. This will more accurately reflect the meaning of the function.

Reply:
fixed